static/fonts/fonts.css

```diff
@@ -16,3 +16,11 @@
     font-weight: normal;
     font-style: normal;
 }
+
+@font-face {
+  font-family: 'Manjari';
+  src: url('Manjari-Regular.woff2') format('woff2'),
```

Comment:
No fallback font?

Reply:
It will fallback to raleway font. no need for explicit declaration